src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1279 @@
+
+      ! set respiration fractions for fluxes between compartments
+      rf_l1m1 = 1.0_r8 - params_inst%mimics_mge(1)
+      rf_l2m1 = 1.0_r8 - params_inst%mimics_mge(2)
+      rf_s1m1 = 1.0_r8 - params_inst%mimics_mge(3)
+      rf_l1m2 = 1.0_r8 - params_inst%mimics_mge(4)
+      rf_l2m2 = 1.0_r8 - params_inst%mimics_mge(5)
+      rf_s1m2 = 1.0_r8 - params_inst%mimics_mge(6)
+
+      ! vmod = "old" vmod * av  AND  kmod = ak / "old" kmod
+      ! Table B1 Wieder et al. (2015) and MIMICS params file give diff
+      ! ak and av values. I used the params file values.
+      vmod_l1_m1 = params_inst%mimics_vmod(1)
+      vmod_l2_m1 = params_inst%mimics_vmod(2)
+      vmod_s1_m1 = params_inst%mimics_vmod(3)
+      vmod_l1_m2 = params_inst%mimics_vmod(4)
+      vmod_l2_m2 = params_inst%mimics_vmod(5)
+      vmod_s1_m2 = params_inst%mimics_vmod(6)
+      kmod_l1_m1 = params_inst%mimics_kmod(1)
+      kmod_l2_m1 = params_inst%mimics_kmod(2)
+      kmod_s1_m1 = params_inst%mimics_kmod(3)
+      kmod_l1_m2 = params_inst%mimics_kmod(4)
+      kmod_l2_m2 = params_inst%mimics_kmod(5)
+      kmod_s1_m2 = params_inst%mimics_kmod(6)
+      vslope_l1_m1 = params_inst%mimics_vslope(1)
+      vslope_l2_m1 = params_inst%mimics_vslope(2)
+      vslope_s1_m1 = params_inst%mimics_vslope(3)
+      vslope_l1_m2 = params_inst%mimics_vslope(4)
+      vslope_l2_m2 = params_inst%mimics_vslope(5)
+      vslope_s1_m2 = params_inst%mimics_vslope(6)
+      kslope_l1_m1 = params_inst%mimics_kslope(1)
+      kslope_l2_m1 = params_inst%mimics_kslope(2)
+      kslope_s1_m1 = params_inst%mimics_kslope(3)
+      kslope_l1_m2 = params_inst%mimics_kslope(4)
+      kslope_l2_m2 = params_inst%mimics_kslope(5)
+      kslope_s1_m2 = params_inst%mimics_kslope(6)
+      vint_l1_m1 = params_inst%mimics_vint(1)
+      vint_l2_m1 = params_inst%mimics_vint(2)
+      vint_s1_m1 = params_inst%mimics_vint(3)
+      vint_l1_m2 = params_inst%mimics_vint(4)
+      vint_l2_m2 = params_inst%mimics_vint(5)
+      vint_s1_m2 = params_inst%mimics_vint(6)
+      kint_l1_m1 = params_inst%mimics_kint(1)
+      kint_l2_m1 = params_inst%mimics_kint(2)
+      kint_s1_m1 = params_inst%mimics_kint(3)
+      kint_l1_m2 = params_inst%mimics_kint(4)
+      kint_l2_m2 = params_inst%mimics_kint(5)
+      kint_s1_m2 = params_inst%mimics_kint(6)
+
+      ! some of these are dependent on the soil texture properties
+      ! One-time initializations here.
+      ! Time-dep params in subr. decomp_rates_mimics.
+
+      do c = bounds%begc, bounds%endc
+         do j = 1, nlevdecomp
+            ! The parameter values currently in the params files always lead to
+            ! positive values for the expressions below, so we do not
+            ! need to use the max function to limit these expressions.
+            ! We apply the min function on cellclay because we are looping over
+            ! some non-soil columns here that contain cellclay = 1e36.
+            clay = dmin1(100.0_r8, cellclay(c,j))
```

Comment:
@slevisconsulting I need to double check with other MIMICS code, but I think the assumption here is that clay for these equations is a fraction, not a percent.  Assuming this is true, we can add a line where clay =clay /100, but should check it's not being used elsewhere in the code.

Reply:
Yes, this needs to be a fraction (range 0-1), not a percent (range 0-100).   Maybe we should call it `clay_frac` to clarify?

With a quick look, I think this the only place where clay is being used in the MIMICS code.  If this is true, @slevisconsulting can you make this change and test this change for in the 1x1 brazil case?

Reply:
Sounds good. Do you prefer a clean start using cold-start or a restart with existing pools?

Reply:
For now started as cold-start to keep the comparison clean.